src/components/Dialogs/RegionDialog/RegionDialogComponent.tsx

```diff
@@ -24,6 +24,8 @@ export class RegionDialogComponent extends React.Component<{ appStore: AppStore
         }
     };
 
+    private handleFocusClicked = () => this.props.appStore.activeFrame.regionSet.selectedRegion.focusCenter(this.props.appStore.activeFrame);
```

Comment:
since a region only ever belongs to a single frame, why don't we just add the frame to the region constructor instead, and add a class variable `private frame: FrameStore` to the `RegionStore`? then we don't need to pass this in every time

Reply:
Fixed